project/task1/detectFixations.py

```diff
@@ -0,0 +1,29 @@
+import statistics
+from pointClass import Point
+from handleWindowPoints import handleWindowPoints
+
+def detectFixations(rowData, DT, SC):
+    fixationDuration = []
+    fixationCentroid = []
+    i=0
+    while i+SC-1 <= len(rowData)-1:
+        D = handleWindowPoints(rowData[i:i+SC])[0]
+        compareDispersion = D > DT
+        if compareDispersion:
```

Comment:
Do we really need to create this intermediate variable?

Reply:
Will change